### PR TITLE
Quiz form (to be continued) + new classroom bundle + Answer/Question entities replacement

### PR DESCRIPTION
**Pull request by sergejvoronov**

### CODE REVIEW

File: src/MasterPeace/Bundle/BookBundle/Factory/QuestionFactory.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+use MasterPeace\Bundle\BookBundle\Entity\Question;
+
+class QuestionFactory
+{
+    const ANSWER_COUNT = 4;
+
+    /**
+     * @return Question
+     */
+    public static function create()
```
Comment (povils) [inline src/MasterPeace/Bundle/BookBundle/Factory/QuestionFactory.php]: kokia prasme sita tureti?

File: src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+     *
+     * @return Response
+     */
+    public function createAction(Request $request)
+    {
+        $quiz = new Quiz();
+        $form = $this->createForm(QuizType::class, $quiz);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
```
Comment (povils) [inline src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php]: nereikis isSubmmiter

File: src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+     * @Route("/edit/{id}", name="quiz_edit")
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function editAction(Request $request, $id)
+    {
+        $quiz = $this->getQuizOr404($id);
+     var_dump($quiz->getQuestions()->toArray()[0]->getAnswers()->toArray()); die;
```
Comment (povils) [inline src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php]: per nagus uz toki:D

File: src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        $form = $this->createForm(QuizType::class, $quiz);
+        $form->handleRequest($request);
+
+        if ($form->isSubmitted() && $form->isValid()) {
```
Comment (povils) [inline src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php]: nereik submiiter

File: src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php (hunk 2 above)
Comment (tadcka) [inline src/MasterPeace/Bundle/QuizBundle/Controller/QuizController.php]: pasiskaitom: https://github.com/symfony/symfony/blob/3.2/UPGRADE-3.2.md#form

File: src/MasterPeace/Bundle/BookBundle/Form/QuestionType.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('title', TextType::class, [
+                'label' => 'Question',
+            ])
+            ->add('answers', CollectionType::class, [
+                'entry_type' => AnswerType::class,
+                'allow_add' => true,
+//                'allow_delete' => true,
```
Comment (tadcka) [inline src/MasterPeace/Bundle/BookBundle/Form/QuestionType.php]: remove :)

File: src/MasterPeace/Bundle/ClassroomBundle/Repository/ClassroomStudentRepository.php [hunk 6]
```diff
@@ -0,0 +1,9 @@
+<?php
+
+namespace MasterPeace\Bundle\UserBundle\Repository;
```
Comment (tadcka) [inline src/MasterPeace/Bundle/ClassroomBundle/Repository/ClassroomStudentRepository.php]: namespace blogas

**Pull request merged**